P/PETSc/build_tarballs.jl

```diff
@@ -54,6 +54,38 @@ build_petsc()
     else
         USE_INT64=0
     fi
+
+    # A SuperLU_DIST build is (now) available on most systems, but only works for double precision
+    USE_SUPERLU_DIST=0    
+    if [[ "${1}" == "double" ]]; then
```

Comment:
Instead of looking at settings and current architecture, you could look for an include file or a library file that SuperLU_DIST would provide. Then you see right away whether it is present, and can enable it in this case. This would be future-proof, in the sense that future changes to SuperLU_DIST would be picked up automatically.

Reply:
thanks, added that